README.adoc

```diff
@@ -293,13 +293,13 @@ EventEntityTest.java
 include::finish/backendServices/src/test/java/it/io/openliberty/guides/event/EventEntityTest.java[tags=!copyright;]
 ----
 
-The [hotspot=54-57]`testInvalidRead()`, [hotspot=60-64]`testInvalidDelete()` and [hotspot=67-71]`testInvalidUpdate()` methods use a primary key that is not in the database to test reading, updating and deleting an event that does not
+The [hotspot=testInvalidRead]`testInvalidRead()`, [hotspot=testInvalidDelete]`testInvalidDelete()` and [hotspot=testInvalidUpdate]`testInvalidUpdate()` methods use a primary key that is not in the database to test reading, updating and deleting an event that does not
 exist, respectively.
 
-The [hotspot=74-87]`testIndividualRead()` method persists a test event to the database and retrieves the 
+The [hotspot=testIndividualRead]`testIndividualRead()` method persists a test event to the database and retrieves the 
```

Comment:
Hotspot is named `testIndividualRead` but references a method named `testReadIndividualEvent`
![image](https://user-images.githubusercontent.com/26514093/66072747-a2d51380-e523-11e9-8ccc-3a88077373dd.png)